dist/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <header>
+      <h1>Leaderboard</h1>
+    </header>
+    <main>
+      <section id="scores">
+        <div class="title">
+          <h3>Recent scores</h3>
+          <button class="refresh">Refresh</button>
+        </div>
+        <ul class="container">
+          <li>Name:100</li>
+          <li>Name:20</li>
+          <li>Name:50</li>
+          <li>Name:78</li>
+          <li>Name:125</li>
+          <li>Name:77</li>
+          <li>Name:42</li>
+        </ul>
```

Comment:
- [ ] good job making your project look like the wireframe 👍 however according to the requirements, you should use HTML just to make the layout, to view the items inside the list you are expected to use Javascript/ES6 modules, with import and export. as mentioned in [the assignment](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md)